style.css

```diff
@@ -13,6 +13,50 @@
   min-height: 650px;
 }
 
+#mobile-menu {
+  display: none;
+  position: fixed;
+  top: 0;
+  right: 0;
+  width: 100%;
+  max-width: 375px;
+  height: 655px;
+  background-color: #3c3a39;
+  color: #fff;
+  z-index: 100;
+}
```

Comment:
Please ensure that the hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (The mobile menu should have full-width like the Figma design).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/161674080-8aa1c493-d08e-4b3e-bb3c-cdcfdf550a24.png) | ![image](https://user-images.githubusercontent.com/10332499/161674113-45b92203-08e9-4c12-8819-2c9740ed342c.png)